styles/pages/settings.module.scss

```diff
@@ -0,0 +1,94 @@
+.main {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	gap: 100px;
+
+	article {
+		display: flex;
+		justify-content: center;
+		align-items: center;
+		gap: 10px;
+
+		h2 {
+			font-weight: 400;
+			font-size: 1.65rem;
+			color: var(--secondary-color);
+		}
+
+		.theme-switch {
```

Comment:
relates to what I said earlier but this should move to a separate module

Reply:
noted